vtex-apps/b2b-organizations
# Feat/organizations search criteria

### PR Description

#### What problem is this solving?

This PR addresses two key updates to improve the user experience in the Organizations module:  
1. The placeholder text in the search input has been updated from **"Search"** to **"Search for ID, trade name or legal name"**, making the purpose of the search field clearer.  
2. The generic alert for unavailable "Trade Name" search functionality has been replaced with a contextual error message:  
   *"The 'Trade Name' search feature is disabled for this account. To enable the search, please set the 'Trade Name' field as 'Is Filterable' in the Masterdata."*  

These changes aim to align the interface with user expectations and reduce confusion while navigating the Organizations search.

#### How to test it?

1. Go to the **Organizations module** in the linked workspace.
2. Verify that the placeholder in the search input reflects the new text:  
   *"Search for ID, trade name or legal name"*.  
3. Trigger the error by attempting to search with a disabled "Trade Name" feature. Confirm the error message matches the provided description.  




[[Workspace](Link goes here!)](https://josmarsoares--b2bstore005.myvtex.com/admin/b2b-organizations/organizations#/organizations)

#### Screenshots or example usage:

https://github.com/user-attachments/assets/e83fdd37-c3f0-49ba-9cd2-b3fd9f128c50






#### Describe alternatives you've considered, if any.



#### Related to / Depends on
- Task: **Organizations Search Criteria Preparation**

#### How does this PR make you feel? [[:link:](http://giphy.com/)](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExZzdmdW5kNXo2bmM5ZzgxODVkNzE1OG1rczEybGdsM3A2b2VkeXdhZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ZubZqIeSsZ60t0ID9l/giphy-downsized-large.gif)

![](put .gif link here - can be found under "advanced" on giphy)
